cupy/core/core.pyx

```diff
@@ -2440,7 +2440,8 @@ cpdef ndarray concatenate_method(tup, int axis):
     ndim = -1
     dtype = None
     have_same_types = True
-    for o in tup:
+    arrays = list(tup)
+    for o in arrays:
```

Comment:
Is this because that `list` is faster than `tuple`?

Reply:
Because `_concatinate` receives list.

Reply:
i see